contrib/devtools/README.md

```diff
@@ -40,6 +40,12 @@ would be changed to:
 
 ```// Copyright (c) 2009-2015 The Bitcoin Core developers```
 
+gen-manpages.sh
+===============
+
+A small script to automatically create manpages in ../../doc/man by running the release binaries with the -help option.
+This requires help2man which can be found at: https://www.gnu.org/software/help2man/
```

Comment:
Hmm, the script was already a bit ugly, but it was very short which was the good thing about it :)
Would you like something like this better: http://pastebin.com/j6JFaiaq

Or I could leave it like it is and write:
>A small script to automatically create manpages in ../../doc/man by running binaries in $PATH with the --help option.